projects/driver_controls/inc/can_output.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+// Module for broadcasting CAN messages
+#include <stdint.h>
+#include "status.h"
+
+// Device ID definitions
+typedef enum {
+  CAN_OUTPUT_DEVICE_ID_POWER = 0,
+  CAN_OUTPUT_DEVICE_ID_PEDAL,
+  CAN_OUTPUT_DEVICE_ID_DIRECTION_SELECTOR,
+  CAN_OUTPUT_DEVICE_ID_TURN_SIGNAL,
+  CAN_OUTPUT_DEVICE_ID_HAZARD_LIGHT,
+  CAN_OUTPUT_DEVICE_ID_MECHANICAL_BRAKE,
+  CAN_OUTPUT_DEVICE_ID_HORN,
+  CAN_OUTPUT_DEVICE_ID_PUSH_TO_TALK,
+  NUM_CAN_OUTPUT_DEVICES
+} CANOutputDeviceID;
```

Comment:
Why is this called "device ID"? These don't seem to be devices on the bus.

Reply:
They were supposed to refer to the input device that triggered the CAN message

Reply:
I hope you can see how that would be confusing.